docs/plugins/repository-azure.asciidoc

```diff
@@ -187,6 +187,19 @@ client.admin().cluster().preparePutRepository("my_backup_java1")
     ).get();
 ----
 
+[[repository-azure-global-settings]]
+==== Global Settings
+
+You can set the number of threads used for repository operations via the following setting:
```

Comment:
I think we should not document this here, as I would like avoid users to play with these knobs, with 32 already being a generous default. The standard rules for configuring any threadpool apply to this one as well.

Reply:
Removed it :)